modules/nf-core/biscuit/pileup/main.nf

```diff
@@ -28,6 +28,7 @@ process BISCUIT_PILEUP {
     if ( tumor_bam.toList().size() > 1 ) error "[BISCUIT_PILEUP] error: more than one tumor BAM provided"
     input = ( tumor_bam==[] ) ? "${normal_bams}" : "-S -T ${tumor_bam} -I ${normal_bams}"
     """
+    set -o pipefail
```

Comment:
nf-core pipelines already have this, cf https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config#L175